src/Forms/Shared/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.xaml

```diff
@@ -7,9 +7,9 @@
   <Grid>
     <Grid.RowDefinitions>
       <RowDefinition Height="*" />
-      <RowDefinition Height="auto" />
+      <RowDefinition Height="100" />
```

Comment:
Lets bump up the height of the listbox to say 150 so that you can see all three line items without having to scroll.